File: RegisterLogin/src/main/java/com/example/RegisterLogin/Dto/DoctorloginDto.java
package com.example.RegisterLogin.Dto;

public class DoctorloginDto {
private String email;
private String password;

public DoctorloginDto() {
}

public DoctorloginDto(String email, String password) {
this.email = email;
this.password = password;
}

public String getEmail() {
	return email;
}

public void setEmail(String email) {
	this.email = email;
}

public String getPassword() {
	return password;
}

public void setPassword(String password) {
	this.password = password;
}

@Override
public String toString() {
	return "DoctorloginDto [email=" + email + ", password=" + password + "]";
} 


}

File: DotnetBackend/kiddoshielddotnet/DTOs/DoctorLoginDto.cs
namespace doctorLoginDto;
public class DoctorloginDto
{
    public string Email { get; set; }
    public string Password { get; set; }

    public DoctorloginDto()
    {
    }

    public DoctorloginDto(string email, string password)
    {
        Email = email;
        Password = password;
    }

    public override string ToString()
    {
        return $"DoctorloginDto [Email={Email}, Password={Password}]";
    }
}
